less/navigation.less

```diff
@@ -1 +1,41 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+.nav-bar {
+    max-width: 1000px;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    margin: 10px auto 0 auto;
+    padding-bottom: 20px;
+    justify-content: space-around;
+    border-bottom: 2px dashed @navigation-border;
+    nav {
+        // display: flex;
+        // flex-direction: row;
+        margin-top: 20px;
+    }
+    a {
+        color: black;
+        width: 35%;
+        font-size: 1.2rem;
+        padding: 20px;
+        align-items: center;
+        text-decoration: none;
+    }
+    h1{
+        width: 35%;
```

Comment:
similarly to the a tags, you probably won't want a width directly on an h1. I'd take this off, with the above flex it shouldn't change much except allow it to be full size in mobile